search-admin/config/deploy.rb

```diff
@@ -0,0 +1,25 @@
+set :application, "search-admin"
+set :capfile_dir, File.expand_path('../', File.dirname(__FILE__))
+set :server_class, "backend"
+
+set :run_migrations_by_default, true
+
+load 'defaults'
+load 'ruby'
+load 'deploy/assets'
+
+set :source_db_config_file, 'to_upload/database.yml'
+
+set :config_files_to_upload, {
+  'to_upload/secrets.yml' => 'config/secrets.yml'
```

Comment:
As per this randomly chosen example of an app with a database configured in here (https://github.com/alphagov/govuk-app-deployment/pull/12/commits/95594abb6d2e939a3b90aa73804d7c38fcef4b87), I think this might need to be `secrets/to_upload/...` both here and on line 11?